frog/imports/ui/Dashboard/MultiWrapper.jsx

```diff
@@ -127,7 +128,24 @@ const MultiWrapper = (props: {
     return <CircularProgress />;
   }
   const aT = activityTypesObj[activity.activityType];
-  const dashNames = names || Object.keys(aT.dashboards || {});
+  const aTDash = aT.dashboards;
+  if (!aTDash || (isEmpty(aTDash) && !names)) {
+    return null;
+  }
+
+  const dashNames =
+    names ||
+    Object.keys(aTDash).filter(name => {
+      const cond = aTDash[name].displayCondition;
+      if (!cond) {
+        return true;
+      }
+      if (typeof cond === 'string') {
+        return !!activity.data[cond];
+      }
+      return cond(activity.data);
```

Comment:
Do you want to add a try/catch around this? As it's code from an external activity package it will not always be implemented by one of us and we don't want it to crash the whole app.

Reply:
We can't put try catch everywhere, and this is just based on config, so it should be very easy to test in preview. We don't do try catch for configUI, which is much more common.

Reply:
👌